backend/apis/events/get-event.api.js

```diff
@@ -26,9 +26,7 @@ app.get("/api/events/:id", (req, res, next) => {
     }
 
     if (event) {
-      res.send({
```

Comment:
The API for GET /api/events/:id does not specify a json response with an event property at the top level.

Instead of
```json
{
  "events": {
    "id": 1
  }
}
```

it's just
```json
{
  "id": 1
}
```